README.md

```diff
@@ -1,51 +1,83 @@
+
+A longer explanation is available in [this blog post](https://eamonn.org/video-gestalt-one-glance-overview-of-a-video).
+
 ## Installation
 
-So far this has only been tested on Linux and Chrome OS, but it will likely work on MacOS too, and maybe even on Windows.
+So far this has been tested on Linux, Chrome OS and Windows, but it will likely work on MacOS too.
 
-The only file you need from this repo is `gestalt.py`. You can grab this however you want, and make it executable. For example, do the following from the command line:
+To install, simply use `pip`:
 
 ```bash
-wget https://raw.githubusercontent.com/eobrain/videogestalt/main/gestalt.py
-chmod +x gestalt.py
+pip install --upgrade videogestalt
 ```
 
-If they are not already installed, you will need to install `python3` and the corresponding Python package manager `pip`.
-
+If they are not already installed, you will need to install `python3` and the corresponding Python package manager `pip` beforehand.
 
 On Linux and friends you might be able to do this like so:
 ```bash
 sudo apt-get install python3 python3-pip
 ```
 
-You will need to install the `moviepy` Python library:
+## Usage
+
+An executable binary `videogestalt` is automatically installed in the local environment.
+
+The following examples assume you cloned the repository to get access to the `example` folder.
+
+To generate a video file:
+
+```bash
+videogestalt -i example/test.mp4 -o test-gestalt --video
+```
+
+To generate an animated GIF (warning, output can be large):
 
 ```bash
-pip install moviepy
+videogestalt -i example/test.mp4 -o test-gestalt --gif
 ```
 
+The application can also be used as a Python module:
 
-## Usage
+```python
+>>> from videogestalt import videogestalt as vg
+>>> vg.main('-i example/test.mp4 -o test-gestalt --gif')
+```
 
-Put the `gestalt.py` in the same directory as an input video file `test.mp4`.
+## Building
 
-Generate a video file:
+The module can be built with PEP517 standard tools, such as `pypa/build`:
 
 ```bash
-./gestalt.py -i test.mp4 -v
+python -sBm build .
 ```
 
-Generate an animated GIF (warning, can be large):
+It can also be installed in development/editable mode after cloning this git repository:
 
 ```bash
-./gestalt.py -i test.mp4 -g
+pip install --upgrade -e .
 ```
 
-[1]: https://user-images.githubusercontent.com/179320/226146985-d67db97e-bcd6-4377-a1da-cc6020135d84.gif
-[3]: https://ia904607.us.archive.org/11/items/vespa-scooter-commercial/Vespa%20Scooter%20Commercial.mp4
+## License
+
+Created by Eamonn O'Brien-Strain.
+
+Licensed under the Mozilla Public License 2.0
+
+[1]: https://github.com/eobrain/videogestalt/master/resources/vespa-commercial-gestalt.gif
```

Comment:
Yes usually when an image is meant to be included in the README and not just in the static website, I prefer to include them in the repository to be more future proof, as URLs of static sites may change or be down at any point in the future (it already happened in the past). There is also another pragmatic reason, it's because I don't know where the image is stored currently (I cannot find the source branch from which the image is pushed to GitHub pages?).